common/content/finder.js

```diff
@@ -274,6 +290,18 @@ var Finder = Module("finder", {
             function () { finder.clear(); },
             { argCount: "0" });
     },
+    events: function() {
+        if (config.name == 'Vimperator') {
+            // Same thing as the note above for not showing the findbar on failed results
+            // through the command line, but for compatibility with FindBar Tweak.
+            events.addSessionListener(window, "WillOpenFindBar", function (event) {
```

Comment:
FBT expects that when the original ```open``` method always leaves the findbar unhidden, which it won't because of the change above.

But I see your point, and thinking about it further it's probably better that I just add checks in FBT to see if the findbar actually opened or not after that was called. I'll upload a commit later today removing this part.